openpaas-james/apps/distributed/pom.xml

```diff
@@ -119,10 +123,6 @@
             <groupId>${james.groupId}</groupId>
             <artifactId>james-server-guice-data-ldap</artifactId>
         </dependency>
-        <dependency>
-            <groupId>org.apache.james</groupId>
-            <artifactId>james-server-guice-distributed</artifactId>
```

Comment:
No this one is perfectly fine: it is not a product but Guice modules for James as a toolkit...